src/FileFormats/NL/NL.jl

```diff
@@ -781,29 +781,29 @@ function _assert_has_model(::Nothing, attr)
     )
 end
 
-_assert_has_model(::MOI.Utilities.UniversalFallback, ::Any) = nothing
+_assert_has_model(model::MOI.Utilities.UniversalFallback, ::Any) = model
 
 function MOI.get(model::Model, attr::MOI.AbstractModelAttribute)
-    _assert_has_model(model.model, attr)
-    return MOI.get(model.model, attr)
+    inner = _assert_has_model(model.model, attr)
+    return MOI.get(inner, attr)
```

Comment:
Previously compiler couldn't prove that `model.model` was not `nothing` because the check happened in `_assert_has_model`. Okay at runtime, not at compilation.